### PR TITLE
Met v3

### PR DESCRIPTION
**Pull request by bmessenger**
Latest approved version "v3" for metronome.io.

### CODE REVIEW

File: public/about/index.html_HOLD [hunk 1]
```diff
@@ -0,0 +1,854 @@
...
+    </footer>
+    <!-- /FOOTER -->
+    <!-- SCRIPTS -->
+    <script src="../dist/js/main.js?cb=1564769302430"></script>
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=%3C%-htmlWebpackPlugin.options.gaTrackingId%%3E"
+    ></script>
+  </body>
+</html>
```
Comment (pablen) [inline public/about/index.html_HOLD]: Are these `*.html_HOLD` files temporary? if so, please remove them.

File: README.md [hunk 2]
```diff
@@ -1,50 +0,0 @@
...
-
-To develop the **static** part of the site with live-reload when files under `/public` or `/src-static` change, run:
-
-```bash
-npm run start:static
-```
-
-## License
-
-MIT
```
[the remark this answers is not in the document]
Comment (bmessenger) [reply README.md]: readme added back

File: public/.htaccess [hunk 3]
```diff
@@ -0,0 +1,5 @@
+AuthUserFile /home/metronome/.htpasswd
+AuthGroupFile /dev/null
+AuthName "Secured Website"
+AuthType Basic
+require user met
```
[the remark this answers is not in the document]
Comment (bmessenger) [reply public/.htaccess]: htaccess removed

File: public/about/index.html_HOLD (hunk 1 above)
Comment (bmessenger) [reply public/about/index.html_HOLD]: unused files removed

**Pull request merged**